jaxopt/_src/implicit_diff.py

```diff
@@ -120,29 +120,124 @@ def root_jvp(optimality_fun: Callable,
   return solve(matvec, Jv)
 
 
-def _custom_root(solver_fun, optimality_fun, solve, has_aux):
-  def solver_fun_fwd(init_params, *args):
-    res = solver_fun(init_params, *args)
-    return res, (res, args)
+def _extract_kwargs(kwarg_keys, args):
```

Comment:
Maybe rename `args` above to `flat_args` (which is consistent with below) or `all_args`. I wasn't sure in the beginning if `args` referred to positional-only arguments or to all arguments.

Reply:
Good call. Done.